querybook/server/app/datasource.py

```diff
@@ -56,6 +61,11 @@ def wrapper(fn):
         @flask_app.route(r"%s%s" % (DS_PATH, url), methods=methods)
         @functools.wraps(fn)
         def handler(**kwargs):
+            # increment the number of api request counter
+            stats_logger.incr(API_REQUEST_COUNTER.format(fn.__name__))
```

Comment:
should we use the url here? it is generic right

Reply:
url is something like `/query_execution/<int:query_execution_id>/`, which will include the parameter types.